### ISSUE TITLE
build(openapi): phase 1 - generate kotlin clients

### ISSUE THREAD

**Issue by petermetz** [hyperledger/cactus#2000]
This is phase one of the task.
All the details are in the phase 2 task's description which you can see here: https://github.com/hyperledger/cactus/issues/468

**Comment by jagpreetsinghsasan** [comment]
I want to work on this

**Comment by jagpreetsinghsasan** [comment]
The attached issues says

The first phase (https://github.com/hyperledger/cactus/issues/2000) is to do what's being described here only for the Corda v4 connector (`./packages/cactus-plugin-ledger-connector-corda/src/main/json/openapi.json`)

I didnt understand what shall be done for the `phase 1`

**Comment by jagpreetsinghsasan** [comment]
Hi @petermetz , any updates on this?

**Comment by jagpreetsinghsasan** [comment]
Understood.

**Comment by petermetz** [comment]
> Hi @petermetz , any updates on this?

@jagpreetsinghsasan Sorry for the slow response. 

> I didnt understand what shall be done for the phase 1

As in, you didn't understand the one (or all) of the tasks in the phase 2 issue description, or something else? I can help either way but the question is too broad like this. ;-)

Also: Make sure to assign the issue to yourself if you are working on it, helps with tracking who works on what.

**Comment by jagpreetsinghsasan** [comment]
Earlier I didn't understand what we need to do in phase 1, given phase2 is for all the plugins. Then I re-read to know that in phase 1, the artifacts should be generated for the corda connector only.

I didn't assign the story straightaway, as I was not sure if this task was important at the moment or not, so I wrote `I want to work on this` and waited for your confirmation. Then later, once I started working on this, I forgot to assign it (Hope no one else worked on this in the meanwhile)
